### PR TITLE
[@types/unl-core] Update types for unl-core to v2.1.x

### PR DESCRIPTION
**Pull request by jake-unl**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [UNL Polyhash library & README](https://github.com/u-n-l/core-js/tree/v2.0/src/polyhash)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## unl-core ([<kbd>unpkg</kbd>](https://unpkg.com/browse/unl-core@latest/))
was missing the following properties:
1. base32
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 43d7662dfc73c57d19ac45b8cb3df34c74f77a61
</p>

File: types/unl-core/index.d.ts [hunk 1]
```diff
@@ -24,20 +24,21 @@ export interface Point {
     lon: number;
 }
 
+export interface PointWithElevation extends Point {
```
Comment (cristian-calugar) [inline types/unl-core/index.d.ts:27]: Has this changed? If yes then why PointWithElevation. extends Point and still defines lat an lon?

File: types/unl-core/index.d.ts [hunk 2]
```diff
@@ -24,20 +24,21 @@ export interface Point {
...
+    elevation: number;
+    elevationType: ElevationType;
+}
+
 export interface Bounds {
     sw: Point;
     ne: Point;
 }
 
 export interface BoundsWithElevation extends Bounds {
```
Comment (cristian-calugar) [inline types/unl-core/index.d.ts:39]: Has this changed?  If yes, then it should not extend Bounds anymore.

File: types/unl-core/unl-core-tests.ts [hunk 3]
```diff
@@ -1,21 +1,100 @@
...
+ *        'u176p0e', '3yyj',
+ *        'zh3',    '72',
+ *        'ey',    'uq',
+ *        '9bjh',    '3zvc',
+ *        'y9',    'z0',
+ *        '6p0e'
+ *     ]
+ * }]
+ */
+const polyGordel = UnlCore.toPolyhash(grachtenGordel, 7);
```
Comment (cristian-calugar) [inline types/unl-core/unl-core-tests.ts:53]: we need to specify the type for polyGordel to validate the return type of the called method. Same for other tests.

**Review by cristian-calugar — APPROVED** [review]

**Comment by jake-unl** [comment]
Ready to merge

**Pull request merged**